--- configure.ac~	2007-04-16 21:29:29.000000000 +0100
+++ configure.ac	2008-09-29 21:44:47.000000000 +0100
@@ -48,7 +48,7 @@
    FORK_FLAG="-DFORK_FLAG=1"
 fi
 
-AC_SUBST([ FORK_FLAG ])
+AC_SUBST([FORK_FLAG])
 
 
 dnl disable xbindkeys_show install
@@ -77,7 +77,7 @@
 
 AM_CONDITIONAL(GUILE, test x$use_guile = xyes)
 
-AC_SUBST([ GUILE_FLAG ])
+AC_SUBST([GUILE_FLAG])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_TYPE_PID_T
